README.md

```diff
@@ -1,52 +1,64 @@
+download an image or package that was just built in the build service, you won't find that in the mirrors, yet.
+- If the image or package has not changed its name, mirrors do not update that file. For example, if you have images
+without the build number, which is very convenient to have (aka static links), new builds of the image are not
+propagated to the mirrors. The same happens for RPMs, for example, if you remove a project and create it again,
+the build number is going to reset and then you can easily have an RPM that has the same build number as before,
+meaning the filename is the same and so the mirror will not get updated.
+- Sometimes metadata and RPMs do not match. If you hit an "update window time", it could be that the metadata has been
+updated on that mirror but it has not finished downloading the RPM.
+- Even zypper has some code to manage metalinks, terraform libvirt does not, so if an image is not found close to your
+location, it is not found at all.
 
 For all that, we had to skip all the openSUSE mirror infrastructure and provide alternative links.
 
-For Pull Requests we have this alternative urls:
-
-* http://jenkins-worker-prs.mgr.prv.suse.net/workspace/suma-prX/repos/ : For every environment, there is a list of repos that are needed for testing a particular Pull Request. This includes the master repo, the pull request repo, the client repos and some other repos that are also needed. All those repos are synced using the Build Service API, instead of from download.opensuse.org.
-
-* http://minima-mirror.mgr.prv.suse.net/jordi/dummy/ : This is an empty repo used instead of the update repos. This way, the build is reproduceable.
-
-* http://minima-mirror.mgr.prv.suse.net/repositories/systemsmanagement:/sumaform:/images:/libvirt/images/ . openSUSE image for testing Pull Requests built with the open build service. This needs to be copied manually.
-
-* http://minima-mirror.mgr.prv.suse.net/distribution/leap/: oss and non-oss repositories 
+For Pull Requests we have this alternative URLs:
+
+- http://jenkins-worker-prs.mgr.prv.suse.net/workspace/suma-prX/repos/: For every environment, there is a list of repos
+that are needed for testing a particular Pull Request. This includes the master repo, the pull request repo, the client
+repos and some other repos that are also needed. All those repos are synced using the Build Service API, instead of from
+download.opensuse.org.
+- http://minima-mirror.mgr.prv.suse.net/jordi/dummy/: This is an empty repo used instead of the update repos. This way,
+the build is reproducible.
+- http://minima-mirror.mgr.prv.suse.net/repositories/systemsmanagement:/sumaform:/images:/libvirt/images/. openSUSE
+image for testing Pull Requests built with the open build service. This needs to be copied manually.
+- http://minima-mirror.mgr.prv.suse.net/distribution/leap/: oss and non-oss repositories.
+
+## Used image versions in the test suite
```

Comment:
In the CI only, maybe add "CI" to title. BV is another beast.

Reply:
Makes sense. I added CI to it.